content/learn/vp_agof_config_controller.adoc

```diff
@@ -0,0 +1,223 @@
+---
+menu:
+  learn:
+    parent: Validated patterns frameworks
+title: Using the Controller Configuration collection 
+weight: 25
+aliases: /ocp-framework/agof/
+---
+
+:toc:
+:imagesdir: /images
+:_content-type: ASSEMBLY
+include::modules/comm-attributes.adoc[]
+
+== Understanding the Ansible GitOps Framework (AGOF) installation process
```

Comment:
Avoid using gerunds in titles if it is not the procedure.
you can write:
Ansible GitOps Framework (AGOF) Installation Process or
Overview of the Ansible GitOps Framework (AGOF) Installation Process